app/validators/file_upload_validator.rb

```diff
@@ -0,0 +1,57 @@
+class FileUploadValidator < ActiveModel::Validator
+  MIN_FILE_SIZE = 5 # KB
+  MAX_FILE_SIZE = 20 # MB
+
+  ALLOWED_CONTENT_TYPES = %w[
+    application/msword
+    application/pdf
+    application/rtf
+    application/vnd.ms-excel
```

Comment:
Are we meant to be allowing excel files? It's not listed on the error on the design

Reply:
Civil don't but do allow csv files

Reply:
This was lifted from civil and not reviewed 😄 so there's a mismatch between the what the designs list as acceptable file types and we what we have in the backend. I'll clean it the list to keep it in line with design